app/src/main/res/layout/fragment_profile_user.xml

```diff
@@ -0,0 +1,394 @@
+                android:layout_marginStart="@dimen/separation_border_parent"
+                android:layout_marginTop="30dp"
+                android:layout_marginEnd="@dimen/separation_border_parent"
+                android:background="@drawable/edit_text_background"
+                android:enabled="false"
+                android:hint="@string/password"
+                android:inputType="textPassword"
+                android:padding="5dp"
+                android:textColor="@color/white"
+                android:textColorHint="#6BFFFFFF"
+                android:textSize="14sp"
+                android:textStyle="normal" />
+
+            <ImageButton
+                android:id="@+id/btnEditEmergencyMessage"
+                android:layout_width="25dp"
+                android:layout_height="25dp"
+                android:layout_above="@+id/etEmergencyMessage"
+                android:layout_alignEnd="@+id/etPassword"
+                android:layout_marginEnd="0dp"
+                android:layout_marginBottom="@dimen/separation_edit_button"
+                android:background="#00FFFFFF"
+                android:src="@drawable/ic_edit"
+                app:tint="@color/blue_cute" />
+
+            <EditText
+                android:id="@+id/etEmergencyMessage"
+                android:layout_width="161dp"
+                android:layout_height="65dp"
+                android:layout_below="@+id/etPassword"
+                android:layout_alignParentStart="true"
+                android:layout_alignParentEnd="true"
+                android:layout_marginStart="@dimen/separation_border_parent"
+                android:layout_marginTop="30dp"
+                android:layout_marginEnd="@dimen/separation_border_parent"
+                android:background="@drawable/edit_text_background"
+                android:enabled="false"
+                android:hint="@string/emergency_message"
+                android:inputType="textPersonName|textMultiLine"
+                android:padding="5dp"
+                android:paddingStart="8dp"
+                android:textAlignment="textStart"
+                android:textColor="@color/white"
+                android:textColorHint="#6BFFFFFF"
+                android:textSize="14sp"
+                android:textStyle="normal" />
+
+            <TextView
+                android:id="@+id/tvEmergencyContacts"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/etEmergencyMessage"
+                android:layout_alignParentStart="true"
+                android:layout_marginStart="@dimen/separation_border_parent"
+                android:layout_marginTop="10dp"
+                android:text="@string/your_emergency_contacts"
+                android:textColor="#000000"
+                android:textSize="20sp"
+                android:textStyle="bold" />
+
+            <androidx.cardview.widget.CardView
```

Comment:
We should break this cardview out into another layout file. We generally want to avoid relative layouts nested into one another.

Reply:
It might make sense to move the new contact logic into a separate fragment and then navigate _back_ here after. Right now it looks like you have a few nested relative layouts + a recycler view at the end which can cause problems if a recycler view is nested inside another scroll view.

Reply:
Note: We'll do this in a TODO at a later time.